daemon/cmd/fqdn_test.go

```diff
@@ -131,13 +142,13 @@ func (ds *DaemonFQDNSuite) Benchmark_notifyOnDNSMsg(c *C) {
 	nameManager.Unlock()
 
 	// Initialize the endpoints.
-	endpoints := make([]*endpoint.Endpoint, c.N)
+	endpoints := make([]*endpoint.Endpoint, b.N)
```

Comment:
Not related to this PR, but I think this benchmark has the same issue I pointed out in another PR, namely changing the input size at each run (see https://github.com/cilium/cilium/pull/32013/files#r1584990720, https://github.com/cilium/cilium/pull/32276 and the "Traps for young players" section in https://dave.cheney.net/2013/06/30/how-to-write-benchmarks-in-go).

I can take care of this in a follow up PR, just wanted to raise awareness and double check my findings.

Reply:
Thanks for sharing, I didn't know about this before. Let's take it separately to avoid the scope expansion for this work.